main.tex

```diff
@@ -142,63 +112,49 @@ \section{Introduction}\label{section:introduction}
 a given memory-one strategy; however, whilst~\cite{Press2012} found a way for a
 player to manipulate a given opponent, this work will consider a
 multidimensional optimisation approach to identify the best response to a given
-group of opponents. In particular, this work presents a compact method of
+group of opponents. The outcomes of our work reinforce known results, namely
+that memory-one strategies must be forgiving to be evolutionarily stable~\cite{Stewart2013, Stewart2016}
+and that longer-memory strategies have a certain form of advantage over short memory
+strategies~\cite{Hilbe2017, Pan2015}.
+
+
+In particular, this work presents a compact method of
 identifying the best response memory-one strategy against a given set of
-opponents, and evaluates whether it behaves extortionately, similar to
-zero-determinants. Further theoretical and empirical results of this work
-include:
-
-\begin{enumerate}
-    \item The conditions that ensure a best response memory-one strategy evolutionary
-    robust.
-    \item A well designed framework that allows the comparison of an optimal
-          memory one strategy and a more complex strategy which has a larger
-          memory and was obtained through reinforcement learning
-          techniques~\cite{Harper2017}.
-    \item An identification of conditions for which defection is known to be
-    stable; thus identifying environments where cooperation will not
-    occur.
-\end{enumerate}
-
-\section{The utility}\label{section:utility}
+opponents, and evaluates whether it behaves in a
+zero-determinant way which in turn indicates whether it can be extortionate. This is also done in evolutionary settings. Moreover, we
+introduce a well designed framework that allows the comparison of an optimal
+memory one strategy and a more complex strategy which has a larger memory, and an
```

Comment:
`... has a larger memory. This is used to identify conditions for which defection is stable; thus identifying environments where cooperation will not occur.`